src/main/scala/com/gd/twitterstreaming/ConfigValidator.scala

```diff
@@ -0,0 +1,16 @@
+package com.gd.twitterstreaming
```

Comment:
> twitterstreaming

according to the naming convention, name of project should follow org subdomain, e.g.:"com.gd.**twitteranalytics**"

Reply:
Renamed.

Reply:
Issue fixed in scope of 
https://github.com/ipetrushin-gd/data-engineering-practice/pull/2